src/components/details.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import '../styles/details.scss';
+
+const Details = () => {
+  const selected = useParams();
+  const country = useSelector((state) => state.HomeReducer
+    .filter((item) => item.country === selected.id));
+  const [details] = useState(...country);
+  const [openToday, setOpenToday] = useState(false);
+  const [openTotal, setOpenTotal] = useState(false);
+  const [openAverage, setOpenAverage] = useState(false);
+
+  const handleKeyDown = () => {
+
+  };
```

Comment:
- [OPTIONAL] I see you declared this function but it's not executing anything.  You can remove it to code clean up your code.

Reply:
Hi, sorry, without that function linters were throwing me errors, and I saw that I could make that so linters don't mess the project

Reply:
Thanks for the update @lisandroseia. I see the reason for that function. But  I will advise you to look for other means to work on the linters.

Reply:
yeah, I was thinking on disable that check, but I thought that would be a bad practice or something, but it seems a better option than writing a blank function, thanks for the advice, I really appreciate it!